### PR TITLE
Updates Nov 18

### PR DESCRIPTION
**Pull request by axxroytovu**
A couple of LCI fixes
Should resolve https://github.com/taw/magic-sealed-data/issues/36

### CODE REVIEW

File: data/boosters/lci.yaml [hunk 1]
```diff
@@ -2,15 +2,18 @@
 pack:
   cave_fullart_land: 1
   foil_chance:
-  - sfc_common_with_showcase: 9
+  - sfc_common: 9
```
Comment (fverdoja) [inline data/boosters/lci.yaml:5]: Is `sfc_common` here and at line 7 correct? Shouldn't they be still `sfc_common_with_showcase`?

Comment (axxroytovu) [reply data/boosters/lci.yaml:5]: There are no single faced, common showcase cards in this set: https://mtg.wtf/card?q=e%3ALCI+is%3Asfc+promo%3Aboosterfun+r%3Ac

Comment (fverdoja) [reply data/boosters/lci.yaml:5]: then possibly the `sfc_common_with_showcase` sheet at line 14 should be `sfc_common`? 
Or maybe everything is correct and I still don't fully understand how you parse these yaml :D if that's the case, sorry for the bother...

Comment (axxroytovu) [reply data/boosters/lci.yaml:5]: No yep, that's a good catch I'll fix line 14

**Pull request merged**